free-programming-books.md

```diff
@@ -1698,6 +1698,7 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Professor Frisby’s Mostly Adequate Guide to Functional Programming](https://mostly-adequate.gitbooks.io/mostly-adequate-guide/content/) - Brian Lonsdorf
 * [Programming JavaScript Applications](https://www.oreilly.com/library/view/programming-javascript-applications/9781491950289/)
 * [Robust Client-Side JavaScript](https://molily.de/robust-javascript/) - Matthias Schäfer (HTML, EPUB)
+* [Single page apps in depth](http://singlepageappbook.com/) - Mixu (HTML)
```

Comment:
For root URLs we prefer not to have a trailing slash, could you remove it, please?

```suggestion
* [Single page apps in depth](http://singlepageappbook.com) - Mixu (HTML)
```